app/src/main/res/values/strings.xml

```diff
@@ -10,196 +10,196 @@
-    <string name="no">NO</string>
-    <string name="existing_vpn_confirm">The currently active VPN app will be disconnected. Do you want to proceed?</string>
-    <string name="vpn_setup_failed">VPN setup failed</string>
-    <string name="app_not_found">App "%1$s" not found</string>
-    <string name="apps_loading_please_wait">Apps loading in progress, please wait</string>
+    <string name="http_server_port">Port</string>
+    <string name="receiver_ip_address">IP address</string>
+    <string name="receiver_port">Port</string>
+    <string name="dump_mode">Dumping mode</string>
+    <string name="user_guide">User guide</string>
+    <string name="open_telegram_group">Telegram group</string>
+    <string name="source_code">Copylefted libre source code</string>
+    <string name="yes">Yes</string>
+    <string name="no">No</string>
+    <string name="existing_vpn_confirm">Disconnect the active VPN app and continue?</string>
+    <string name="vpn_setup_failed">Could not set up VPN</string>
+    <string name="app_not_found">Could not find the \"%1$s\" app</string>
+    <string name="apps_loading_please_wait">Loading apps…</string>
     <string name="stats">Stats</string>
-    <string name="active_connections">Active Connections</string>
-    <string name="dropped_connections">Dropped Connections</string>
-    <string name="tot_connections">Total Connections</string>
-    <string name="open_sockets">Open Sockets</string>
+    <string name="active_connections">Active connections</string>
+    <string name="dropped_connections">Dropped connections</string>
+    <string name="tot_connections">Total connections</string>
+    <string name="open_sockets">Open sockets</string>
     <string name="max_file_descriptor">Max FD</string>
-    <string name="bytes_sent">Bytes Sent</string>
-    <string name="bytes_rcvd">Bytes Received</string>
-    <string name="packets_sent">Packets Sent</string>
-    <string name="packets_rcvd">Packets Received</string>
-    <string name="dns_queries">DNS Queries</string>
-    <string name="search_apps">Search Apps</string>
+    <string name="bytes_sent">Bytes sent</string>
+    <string name="bytes_rcvd">Bytes received</string>
+    <string name="packets_sent">Packets sent</string>
+    <string name="packets_rcvd">Packets received</string>
+    <string name="dns_queries">DNS queries</string>
+    <string name="search_apps">Search…</string>
     <string name="no_apps">No apps</string>
     <string name="dns_server">DNS Server</string>
     <string name="apps">Apps</string>
-    <string name="navigation_drawer_open">Drawer Open</string>
-    <string name="navigation_drawer_close">Drawer Close</string>
-    <string name="pcap_file">PCAP File</string>
-    <string name="pcap_file_info">Create a PCAP file into the device storage</string>
-    <string name="cannot_write_file">Cannot write file</string>
+    <string name="navigation_drawer_open">Open drawer</string>
+    <string name="navigation_drawer_close">Close</string>
```

Comment:
The idea is that when the drawer is already open, "Close" is sufficient.
I couldn't find this anywhere though.